lib/Competition/Competition.cpp

```diff
@@ -117,20 +127,63 @@ bool Competition::handleCompetition(String &outputMessage)
     return isSuccess;
 }
 
-bool Competition::setReleasedState()
+bool Competition::setReleasedState(const uint8_t &activeGroup)
 {
     bool isSuccess = false;
 
-    if ((COMPETITION_STATE_UNRELEASED == m_competitionState) ||
-        (COMPETITION_STATE_FINISHED == m_competitionState))
+    if (MAX_NUMBER_OF_GROUPS - 1 > activeGroup)
```

Comment:
Use braces to make explicit which step shall be done first. :-)

Reply:
I am not sure how/where the braces should go

Reply:
```((MAX_NUMBER_OF_GROUPS - 1)  > activeGroup)```

Reply:
Oh.......